servers/src/main/java/tachyon/master/MasterBase.java

```diff
@@ -41,6 +44,8 @@
 public abstract class MasterBase implements Master {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
 
+  private static final long SHUTDOWN_TIMEOUT_MS = 10000;
```

Comment:
This doesn't affect how soon master shuts down - it just specifies how we're willing to wait before we give up on shutdown succeeding. I don't think this needs to be configurable because it would indicate a bug if it ever took close to this long.